src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/custom/DSLParamStrategy.java

```diff
@@ -25,14 +25,17 @@ public String getOrderedChildrenDSL() {
 		PropertyDescriptor propertyDescriptor = (PropertyDescriptor) getDescriptor();
 		if (propertyDescriptor.getName().equals("hudson.model.StringParameterDefinition") || propertyDescriptor.getName().equals("hudson.model.BooleanParameterDefinition")) {
 			String defaultValue = "\"\"";
+			String description = "\"\"";
 			List <PropertyDescriptor> children = propertyDescriptor.getProperties();
 			for (PropertyDescriptor child : children ) {
 				if (child.getName().equals("defaultValue")) {
 					defaultValue = getChildrenByName("defaultValue").toDSL();
 				}
+				if (child.getName().equals("description")) {
+					description = getChildrenByName("description").toDSL();
+				}
```

Comment:
At this point maybe it's better to loop over each child and assign a default value if it's blank

Reply:
I can find a better way to do this, but the issue is that the child tag doesn't exist at all so it would get to the function,  getChildrenByName.toDsl and throw an exception. I'll try to find a cleaner way to do this

Reply:
Yeah you'll want to get all the children at once, not one at a time. getChildren() or propertyDescriptor.getProperties would do it